core/src/main/resources/error/error-classes.json

```diff
@@ -1130,6 +1135,11 @@
       "Unable to convert SQL type <toType> to Protobuf type <protobufType>."
     ]
   },
+  "UNBOUND_SQL_PARAMETER" : {
+    "message" : [
+      "Found the unbound parameter: <name>. Please, fix `args` and provide a mapping of the parameter to a SQL literal statement."
```

Comment:
Won't this same error be used fro all other APIs (JDBC, SQL (when we support). So we may not want to refer to `args`.

Reply:
> So we may not want to refer to args.

This is a premature generalisation.  Let's make it more generic when we will need that.